package.json

```diff
@@ -70,6 +71,7 @@
     "eslint-config-prettier": "^8.10.0",
     "eslint-plugin-prettier": "^5.0.0",
     "eslint-plugin-unused-imports": "^3.0.0",
+    "monaco-editor": "^0.43.0",
```

Comment:
Do we use this package anywhere?

Reply:
Also we don't need package-lock.json, we use yarn.

Reply:
I added `monaco-editor` because it helped me with intellisense and types. I am taking it out to keep this PR more scoped

Reply:
package-lock added by mistake. Removing it